testing/src/main/java/org/oppia/android/testing/RecyclerViewTesting.kt

```diff
@@ -0,0 +1,27 @@
+package org.oppia.android.testing
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import org.oppia.android.testing.RecyclerViewMatcher.Companion.atPositionOnView
+import javax.inject.Inject
+
+/**
+ * Central utility for testing items present in RecyclerView.
+ */
+class RecyclerViewTesting @Inject constructor() {
+
+  fun verifyItemDisplayedOnRecyclerView(
```

Comment:
How about if we shift this function to `RecyclerViewMatcher` class. This will make `RecyclerViewMatcher` as central utility.

Reply:
Sounds good to me. So, shall I remove this newly added `RecyclerViewTesting` utility and shift this function. Please confirm this too.

Reply:
Yep

Reply:
Done